tre_schemas/dri-preingest-sip-error.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "$id":"https://nationalarchives.gov.uk/da-transform/tre/schemas/dri-preingest-sip-available",
```

Comment:
Should the $id value end `dri-preingest-sip-error` instead of `dri-preingest-sip-available`?